src/main/java/me/karakelley/http/RequestBuilder.java

```diff
@@ -4,7 +4,13 @@
 
 public class RequestBuilder {
 
-  public static Request build(String requestMethod, String requestPath, String requestProtocol, Map<String, String> headers, int port) {
-    return new Request(requestMethod, requestPath, requestProtocol, headers, port);
+  public static Request build(Map<String, String> requestLineAndBody, Map<String, String> headers, int port) {
```

Comment:
I think it would be cleaner, less brittle, and more type safe if you actually implemented the standard "builder pattern" and then `RequestParser` used that instead of `Map<String, String>`.